dcos_installer/upgrade.py

```diff
@@ -96,6 +96,19 @@
 pkgpanda fetch --repository-url={{ bootstrap_url }} {{ cluster_packages }} > /dev/null
 pkgpanda activate --no-block {{ cluster_packages }} > /dev/null
 
+end=$(date -ud "5 minute" +%s)
+while [[ $(date -u +%s) -le $end ]]
+do
+  if dcos-shell 3dt check --check-config {{ dcos_check_runner_config_path }} node-poststart > /dev/null; then
```

Comment:
We're going to have to teach `3dt` to use a default config file in order for the installer to use it, so you can remove the `--check-config` arg  and the `dcos_check_runner_config_path` gen param.